src/features/search/components/SearchListHeader/SearchListHeader.native.test.tsx

```diff
@@ -252,14 +109,21 @@ describe('<SearchListHeader />', () => {
   })
 
   it('should not display the geolocation button if position is not null', () => {
-    render(<SearchListHeader nbHits={10} userData={[]} venuesUserData={[]} venues={mockVenues} />)
+    render(
+      <SearchListHeader nbHits={10} userData={[]} venuesUserData={[]} venues={mockAlgoliaVenues} />
+    )
 
     expect(screen.queryByText('Géolocalise-toi')).not.toBeOnTheScreen()
   })
 
   it('should display the geolocation incitation button when position is null', () => {
-    mockPosition = null
-    render(<SearchListHeader nbHits={10} userData={[]} venuesUserData={[]} venues={mockVenues} />)
+    mockUseLocation.mockReturnValueOnce({
+      geolocPosition: null as unknown as GeoCoordinates,
```

Comment:
Le typage actuel ne permet pas d'utiliser null 😬

Reply:
tu peux remplacer ce cas par le cas ou c'est undefined ? 
`should display the geolocation incitation button when position is bit defined`